packages/api-clients/src/axiosLogInterceptors.ts

```diff
@@ -32,13 +32,21 @@ export function configureAxiosLogInterceptors(
           prefixText: getPrefix(response.config.headers, clientName),
         }
       ) as AxiosResponse,
-    (error: AxiosError): Promise<AxiosError> =>
-      AxiosLogger.errorLogger(
+    (error: AxiosError): Promise<AxiosError> => {
+      const prefixText = getPrefix(error.config?.headers, clientName);
+      if ("errors" in error && Array.isArray(error.errors)) {
+        error.errors.forEach((err) =>
+          genericLogger.error(`[${prefixText}] ${err}`)
+        );
+      }
```

Comment:
Unrelated. Adding better logging in case of Axios AggregateError - e.g., when the server cannot be reached Axios returns an aggregate error and without logging also the errors inside it's hard to understand:
Now it looks like this:
<img width="517" alt="image" src="https://github.com/user-attachments/assets/7fa5fe6a-e337-4ca8-b23e-225bf4764096">